boards/lilygo/ttgo_t7v1_5/doc/index.rst

```diff
@@ -0,0 +1,222 @@
+.. zephyr:board:: ttgo_t7v1_5
+
+Overview
+********
+
+LILYGO® TTGO T7 Mini32 V1.5 ia an IoT mini development board
+based on the Espressif ESP32-WROVER-E module.
+
+It features the following integrated components:
+- ESP32 chip (240MHz dual core, 520KB SRAM, Wi-Fi, Bluetooth)
+- on board antenna
+- Micro-USB connector for power and communication
+- JST GH 2-pin battery connector
+- LED
+
+Functional Description
+**********************
+This board is based on the ESP32-WROVER-E module with 4MB of flash (there
+are models 16MB as well), WiFi and BLE support. It has a Micro-USB port for
+programming and debugging, integrated battery charging and an on-board antenna.
+
+Connections and IOs
+===================
+
+The ``ttgo_t7v1_5`` board target supports the following hardware features:
```

Comment:
needs proper board targets (appcpu and procpu)

Reply:
figured it out, done